PermissionsEx/permissions.yml

```diff
@@ -12441,6 +12441,33 @@ groups:
         - ch.alias.haste1
         - ch.alias.haste2
         - ch.alias.haste3
+ yangxiaolong:
```

Comment:
her name should have 2 spaces in front, not 1

Reply:
Oh hork said only one ok Ill fix it